spec/requests/case_logs_controller_spec.rb

```diff
@@ -755,7 +755,7 @@
 
       it "downloads a CSV file with headers" do
         csv = CSV.parse(response.body)
-        expect(csv.first.first).to eq("id")
+        expect(csv.first.first).to eq("\uFEFFid")
```

Comment:
Might be worth still leaving in:

```ruby
        expect(csv.first.second).to eq("id")
```

Reply:
The second header is "status", the first header is "\uFEFF" + "id" - I don't think it's worth testing the second is it?

Reply:
Ah gotcha, missed that